docs/changelog.md

```
@@ -31,7 +31,9 @@
 
 ## [Unreleased]
 
-### Added- **`spawn_subagent` — accept `command`/`task`/`text` as `instruction` aliases.** The executor model may emit spawn_subagent tool calls with command/task/text field names instead of instruction, causing 'missing 1 required keyword-only argument: instruction' on the CEO and any delegating agent. The _spawn_subagent() method now promotes any of those three field names to instruction before validating non-emptiness. Fixes CEO error: AgentRunner._spawn_subagent() missing 1 required keyword-only argument: instruction.
+### Added
+- **Autonomous agency maturation — contract enforcement, KPI tracking, trend watcher, log watcher, and e2e test coverage.** New  validates agent tool outputs against Pydantic schemas.  provides thread-safe autonomy KPI tracking (13 metrics).  fetches AI trends from 13 public sources in parallel.  monitors logs and auto-creates GitHub issues. 16 new e2e tests in .
+- ** **`spawn_subagent` — accept `command`/`task`/`text` as `instruction` aliases.** The executor model may emit spawn_subagent tool calls with command/task/text field names instead of instruction, causing 'missing 1 required keyword-only argument: instruction' on the CEO and any delegating agent. The _spawn_subagent() method now promotes any of those three field names to instruction before validating non-emptiness. Fixes CEO error: AgentRunner._spawn_subagent() missing 1 required keyword-only argument: instruction.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)